### ISSUE TITLE
Agora video view cutting off in mobile screen

### ISSUE THREAD

**Issue by devsideal** [AgoraIO-Extensions/Agora-Flutter-SDK#1152]
**What I want**
I need to create full-screen live streaming on mobile.

**What I’m doing**
Using `AgoraVideoView` (without any resizing/modification) as given in the doc or sample.

**What is the issue:**
When I stream from mobile its preview is cutting off from left-right, while on the web (as Audience) its showing correct.

**Steps To Reproduce**
We can reproduce the same issue with the given example too:
https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/blob/main/example/lib/examples/basic/join_channel_video/join_channel_video.dart
1- Clone the Agora-Flutter-SDK
2- Do the little changes in UI to make it full screen: 
2.1-  In `main.dart` Remove the `appBar` from Scaffold under the ListTile 
2.2- In `example_actions_widget.dart`  Remove the bottom padding from Container
3- Run the code and go to JoinChannelVideo screen
4- Join the channel as Host
5- Open the [web-demo](https://webdemo.agora.io/basicLive/index.html) and join same channel as Audience
6- Then you will see the Video preview difference as below screen shots:

**Expected behavior**
The preview should be the same on both ends Mobile (as a host) and Web (as an audience) without any cut-off.

**Screenshots**
### Mobile (as a Host)
![Screenshot_20230615_180344](https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/assets/91250232/441ee3ec-68d1-4ca9-bade-2504479fadd0)

### Web (as an Audience)
<img width="346" alt="Screenshot 2023-06-15 at 6 03 26 PM" src="https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/assets/91250232/ccfd11ef-a0f4-44ee-8c06-0d11084945d3">

**Comment by littleGnAl** [comment]
Can you check our example to see if it works or not?
https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/blob/main/example/lib/examples/advanced/screen_sharing/screen_sharing.dart

**Comment by devsideal** [comment]
@littleGnAl My issue is about Live streaming not with screen sharing, and I'm able to reproduce this with the below example also, please see STR,
https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/blob/main/example/lib/examples/basic/join_channel_video/join_channel_video.dart

**Comment by littleGnAl** [comment]
It is most likely the web demo set the render mode type, you can set the https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/blob/80c9e52b7032b8224d2753c97acaaa3e54389161/lib/src/agora_base.dart#L4319 in the example to see if it meet your need.

**Comment by devsideal** [comment]
Yes, I check there are two types of render modes:
RenderModeType.renderModeFit :  Using this preview not shows on full screen of mobile. 
RenderModeType.renderModeHidden : Using this preview is cutting off from left-right.

**Comment by littleGnAl** [comment]
So I think the `RenderModeType.renderModeFit ` is what you need.

**Comment by devsideal** [comment]
No, `renderModeFit` is not showing the preview on full screen in mobile, I need something that can make the preview according to the device camera, I found there is one more mode `renderModeAdaptive` maybe this is that I want, I need to check this, But this is Deprecated.
